src/lib/ks/interactive

```diff
@@ -134,6 +134,9 @@ if [ -f /etc/nethserver-release ]; then
   echo "Fix /etc/issue..."
   cp -f /etc/nethserver-release /etc/issue
   echo -e 'Kernel \\r on an \\m\n' >> /etc/issue
+  echo 'To login to NethServer please use following URL(s):' >> /etc/issue
```

Comment:
A fixed `NethServer` string could not suit with rebranded products (i.e. NethServer Enterprise). I'd rather write

    Access the web interface at the following URLs:

Reply:
It was intended to work only on first boot so no need for templating or writing directly to console as far as I understand.
I'll change the text to be neutral as you suggested.